Library/Services/Implementation/NfieldLocalUserService.cs

```diff
@@ -0,0 +1,107 @@
+//    GNU Lesser General Public License for more details.
+//
+//    You should have received a copy of the GNU Lesser General Public License
+//    along with Nfield.SDK.  If not, see <http://www.gnu.org/licenses/>.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Nfield.Infrastructure;
+using Nfield.Models;
+
+namespace Nfield.Services.Implementation
+{
+    /// <summary>
+    /// Implementation of <see cref="INfieldSurveysService"/>
+    /// </summary>
+    internal class NfieldLocalUserService : INfieldLocalUserService, INfieldConnectionClientObject
+    {
+        public INfieldConnectionClient ConnectionClient { get; internal set; }
+        public void InitializeNfieldConnection(INfieldConnectionClient connection) => ConnectionClient = connection;
+
+        public async Task<IEnumerable<LocalUser>> GetAllAsync()
+        {
+            var uri = new Uri(ConnectionClient.NfieldServerUri, "LocalUsers");
+
+            using (var response = await ConnectionClient.Client.GetAsync(uri))
+            {
+                return await DeserializeJsonAsync<List<LocalUser>>(response);
+            }
+        }
+
+        public async Task<LocalUser> GetAsync(string identityId)
+        {
+            var uri = new Uri(ConnectionClient.NfieldServerUri, $"LocalUsers/{identityId}");
+
+            using (var response = await ConnectionClient.Client.GetAsync(uri))
+            {
+                return await DeserializeJsonAsync<LocalUser>(response);
+            }
+        }
+
+        public async Task<LocalUser> CreateAsync(NewLocalUser model)
+        {
+            if (model == null)
+                throw new ArgumentNullException(nameof(model));
+
+            var uri = new Uri(ConnectionClient.NfieldServerUri, "LocalUsers");
+
+            using (var response = await ConnectionClient.Client.PostAsJsonAsync(uri, model))
+            {
+                var result = await DeserializeJsonAsync<LocalUser>(response);
+
+                return result;
+            }
+        }
+
+        public async Task<LocalUser> UpdateAsync(string identityId, LocalUserValues model)
+        {
+            if (model == null)
+                throw new ArgumentNullException(nameof(model));
+
+            var uri = new Uri(ConnectionClient.NfieldServerUri, $"LocalUsers/{identityId}");
+
+            using (var response = await ConnectionClient.Client.PatchAsJsonAsync(uri, model))
+            {
+                var result = await DeserializeJsonAsync<LocalUser>(response);
+
+                return result;
+            }
+        }
+
+        public async Task DeleteAsync(string identityId)
+        {
+            var uri = new Uri(ConnectionClient.NfieldServerUri, $"LocalUsers/{identityId}");
+
+            // note: we need to dispose the response even when we don't use it
+            using (await ConnectionClient.Client.DeleteAsync(uri))
+            {
```

Comment:
Is it possible to do it without the brackets {}? Maybe with a simple `;` after the using?

Reply:
.Dispose()?